Atomic/util.py

```diff
@@ -365,7 +365,7 @@ class DockerObjectNotFound(ValueError):
     def __init__(self, msg):
         super(DockerObjectNotFound, self).__init__("Unable to associate '{}' with an image or container".format(msg))
 
-def get_atomic_config():
+def get_atomic_config(ATOMIC_CONF=ATOMIC_CONF):
```

Comment:
This is for overriding test config (see tests/unit/test_trust.py). Throws pylint warning. Any ideas how to do differently are welcome.